python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -788,7 +788,7 @@ def test_compiled_dag_ref_del(ray_start_regular):
     compiled_dag.teardown()
 
 
-def test_dag_fault_tolerance_chain(ray_start_regular_shared):
+def test_dag_fault_tolerance_chain(ray_start_regular):
```

Comment:
When using `ray_start_regular_shared`, Raylet will print a warning message saying, "XX PYTHON worker processes have been started on node: ...". `XX` becomes larger and larger if we use `ray_start_regular_shared`.

<img width="1440" alt="Screenshot 2024-07-29 at 11 31 16 AM" src="https://github.com/user-attachments/assets/022ba15d-bcba-4a1a-bd7c-0dc2dd93634a">

Reply:
hmm this usually indicates the workers are not properly cleaned up.